### DISCUSSION TITLE
Turn off IME mode on some screen

### DISCUSSION THREAD

**Discussion by User-htt-github** [AvaloniaUI/Avalonia#11027] [Q&A]
Is there anyway to turn off IME mode on a window screen cause everytime i type randomly with IME mode is on, it will show up its recommend box on screen like this:
![image](https://user-images.githubusercontent.com/118340304/232384092-b5a591c6-84f4-4e58-9d1c-ca53f7c97fa8.png)

**Comment by Gillibald** [comment]
This is not possible at the moment but not complex to add.

Update: Turns out I misunderstood your question. Currently, IME is initialized on Windows all the time when it should only be initialized when some text input client is present.

**Reply by kekekeks** [reply]
IIRC there was some routed event that was triggered to find the ime client. Is it removed?

**Comment by Gillibald** [comment]
Will be fixed by: https://github.com/AvaloniaUI/Avalonia/pull/11034

**Reply by User-htt-github** [reply]
Thank you
